Updates for April releases / updates for upcoming releases (#1131)
smartdevicelink/sdl_evolution: README.md

## Recent and Upcoming Releases
You can see a detailed list of accepted and implemented proposals for upcoming versions on the [proposal status page][proposal-status].  Each proposal is listed as only impacting one platform, as it's inherent that some platforms impact others, as described above in the `Platforms` Section.

> ##Recent

Reviewer: ```suggestion
## Recent
```

---

#### SDL RPC Spec: 7.1.0
**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/rpc_spec/releases/tag/7.1.0).

---

#### SDL Core: 7.1.0

**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_core/releases/tag/7.1.0).

---

#### SDL iOS: 7.1.0
**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.1.0)

---

#### SDL Java Suite: 5.1.0
**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.1.0)

---

#### SDL JavaScript Suite: 1.3.0
**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_javascript_suite/releases/tag/1.3.0).

---

#### SDL Policy Server: 2.11.0
**Release Date**: April 14, 2021

**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/2.11.0).

---

> ##Upcoming 

Reviewer: ```suggestion
## Upcoming 
```

#### Manticore: 2.8.0
**Planned Release Date**: May 17, 2021

**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/manticore/projects/3). 

---

#### SDL iOS: 7.2.0
**Planned Release Date**: June 30, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_ios/projects/24).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/24).
```

---

#### SDL Java Suite: 5.2.0
**Planned Release Date**: June 30, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.0.0).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/18).
```

---

#### SDL JavaScript Suite: 1.4.0
**Planned Release Date**: June 30, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/5).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/5).
```

---

#### SDL Protocol Spec: 5.5.0
**Planned Release Date**: October 27, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/protocol_spec/projects/6).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/protocol_spec/projects/6).
```

---

#### SDL RPC Spec: 7.2.0
**Planned Release Date**: October 27, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_core/projects/20).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/rpc_spec/projects/7).
```

---
#### SDL Core: 7.2.0
**Planned Release Date**: October 27, 2021

> **Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/rpc_spec/projects/7).

Reviewer: ```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/20).
```
